tests/10-vcdm2.js

```diff
@@ -85,10 +87,12 @@ describe('Contexts', function() {
           // positive @context test
           const vc = await endpoints.issue(require(
             './input/credential-ok.json'));
-          vc.should.have.property('@context');
+          vc.should.have.property('@context',
+            'Failed to respond with a VC with `@context` intact.');
```

Comment:
```suggestion
            'Failed to respond with a VC with intact `@context`.');
```

Reply:
This one's subjective, but if it makes you happy. 😉